src/main/java/behavior/Figure.java

```diff
@@ -0,0 +1,10 @@
+package behavior;
+
+public abstract class Figure extends ColorSupplier {
+    private Colors color;
+
+    protected Figure() {
+    }
+
+    public abstract double getArea();
```

Comment:
should this method be here in the abstract class? let's move this method to the interface